lib/ansible/cli/galaxy.py

```diff
@@ -580,7 +580,11 @@ def exit_without_ignore(rc=1):
     def _display_role_info(role_info):
 
         text = [u"", u"Role: %s" % to_text(role_info['name'])]
-        text.append(u"\tdescription: %s" % role_info.get('description', ''))
+
+        # Get description from galaxy_info['galaxy_info']['description'] first, falling back to top-level 'description'.
+        galaxy_info = role_info.get('galaxy_info', {})
+        description = galaxy_info.get('description', role_info.get('description', ''))
```

Comment:
Just in case people updated roles to contain a top level 'description' key, maybe that should take precedence so the behavior doesn't change.

Reply:
I got a +1 from sivel to update the precedence, so I did that. I forgot to update the comment too - fixed in the last commit. This should be passing tests now and looks good to merge to me.